packages/clangml/clangml.4.4.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for Clang API"
+description: """
+clangml provides bindings to call the Clang API from OCaml.
+"""
+maintainer: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+authors: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+license: "BSD-2-Clause"
+homepage: "https://memcad.gitlabpages.inria.fr/clangml/"
+doc: "https://memcad.gitlabpages.inria.fr/clangml/doc/clangml/index.html"
+bug-reports: "https://gitlab.inria.fr/memcad/clangml/issues"
+depends: [
+  "conf-libclang"
+  "conf-ncurses"
+  "conf-zlib"
+  "dune" {>= "1.11.0"}
+  "stdcompat" {>= "13"}
+  "ocaml" {>= "4.08.0"}
+  "ocamlfind" {build & >= "1.8.0"}
+  "ocamlcodoc" {with-test & >= "1.0.1"}
+  "pattern" {with-test & >= "0.2.0"}
+  "metapp" {>= "0.4.0"}
+  "metaquot" {>= "0.4.0"}
+  "refl" {>= "0.4.0"}
+  "odoc" {with-doc & >= "1.5.1"}
+]
+x-ci-accept-failures: [
+   "oraclelinux-7" "oraclelinux-8" # llvm-config not available
+   "centos-7" # GCC does not support --std=c++14
+   "debian-10" # Extra distributions fail
```

Comment:
I don't think this should be there. There is a genuine segfault:
```
#     ocamlopt clangml/.clang.objs/native/clang__Clang__bindings.{cmx,o} (got signal SEGV)
```
Those platforms (ppc64 and arm32 either have to be forbidden using the `available` field or this issue should be debugged)

Reply:
You are right: I forgot about the `available` field, but it is definitively better than `x-ci-accept-failures` in this case (and I would prefer not to debug these issues for this release).